### PR TITLE
feat(BUILD-5162): ghcr.io Docker images version

### PR DESCRIPTION
**Pull request by julien-carsique-sonarsource**
Locally tested on https://github.com/SonarSource/cirrus-modules

### CODE REVIEW

**Comment by julien-carsique-sonarsource** [comment]
@matemoln I also tried to pin the version with their digest but it was raising a warning, see https://sonarsource.atlassian.net/browse/BUILD-5162?focusedCommentId=625996

File: README.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ load("github.com/SonarSource/cirrus-modules@2.9.0", "load_features")
 load("github.com/SonarSource/cirrus-modules@54babd3268dd6daf42ad877100789169a14e5fb3", "load_features")  # 2.9.0
 ```
 
+#### ghcr.io Docker images in Cirrus CI YAML file
+
+Replaces ghcr.io Docker images version in `.cirrus.yaml`, `.cirrus.yml`.
+
```
Comment (matemoln) [inline README.md:74]: Could you please provide an example?

File: re-team.json [hunk 2]
```diff
@@ -73,12 +73,23 @@
...
             ]
+        },
+        {
+            "customType": "regex",
+            "fileMatch": [
+                "^\\.cirrus\\.ya?ml$"
+            ],
+            "matchStrings": [
+                "image: (?<depName>ghcr.io.*?):(?<currentValue>v?.*)@(?<currentDigest>sha256:[a-f0-9]{64})",
+                "image: (?<depName>ghcr.io.*?):(?<currentValue>v?.*)"
```
Comment (matemoln) [inline re-team.json:90]: The first regex does not match the expected image format. What do you think about merging the 2 expressions like: https://regex101.com/r/ggg0B3/2?

Comment (julien-carsique-sonarsource) [reply re-team.json:90]: My bad, the last commit was missing. sha256 is part of the digest. I'm updating it into a online.

Comment (julien-carsique-sonarsource) [reply re-team.json:90]: https://regex101.com/r/ggg0B3/3

**Review by matemoln — APPROVED** [review]

**Pull request merged**